atom/browser/api/atom_api_web_contents.cc

```diff
@@ -744,6 +744,14 @@ int WebContents::GetID() const {
   return web_contents()->GetRenderProcessHost()->GetID();
 }
 
+std::string WebContents::GetType() const {
+  switch (type_) {
+    case BROWSER_WINDOW: return "window";
+    case WEB_VIEW: return "webview";
+    case REMOTE: return "remote";
```

Comment:
@zcbenz We should never add extensions for any "remote" `webContents` right? It looked like `REMOTE` was used in this file to return early from devtools related methods.